xbmc/rendering/gles/RenderSystemGLES.cpp

```diff
@@ -109,6 +109,11 @@ bool CRenderSystemGLES::InitRenderSystem()
     m_renderCaps |= RENDER_CAPS_BGRA_APPLE;
   }
 
+  if (IsExtSupported("GL_EXT_unpack_subimage"))
+  {
+    m_renderCaps |= RENDER_CAPS_UNPACK_ROW_LENGTH;
```

Comment:
imo those flags are crap in general and adding an additional one makes zero sense. instead of querying the flag you can just query for the extension.